src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -20,7 +20,7 @@
 use ApiPlatform\Core\Util\ClassInfoTrait;
 use Symfony\Component\PropertyAccess\PropertyAccess;
 use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
-use Symfony\Component\Routing\Exception\ExceptionInterface;
+use Symfony\Component\Routing\Exception\ExceptionInterface as RoutingExceptionInterface;
```

Comment:
@theofidry That seems like overkill? :stuck_out_tongue: Let's try to strike a balance...